### PR TITLE
Update terminology.tex

### PR DESCRIPTION
**Pull request by mschauer**

### CODE REVIEW

File: exploration/docs/terminology.tex [hunk 1]
```diff
@@ -127,8 +127,8 @@ \section{Optical Constructions}
 may be written explicitly as
 
 \[
-l_{12}(a) := (x_1 \otimes y_1, y_2) \text { where } x = l_1(a), y = l_2(x_2) \]\[
-r_{12}(m_1 \otimes m_2, c) := r_1(m_1, r_2(m_2, c'))
+l_{12}(a) := (m_1 \otimes m_1, y_2) \text { where } (m_1, b) = l_1(a), (m_2, c) = l_2(b) \]\[
```
Comment (mschauer) [inline exploration/docs/terminology.tex]:
```suggestion
l_{12}(a) := (m_1 \otimes m_2, y_2) \text { where } (m_1, b) = l_1(a), (m_2, c) = l_2(b) \]\[
```

File: exploration/docs/terminology.tex [hunk 2]
```diff
@@ -127,8 +127,8 @@ \section{Optical Constructions}
 may be written explicitly as
 
 \[
-l_{12}(a) := (x_1 \otimes y_1, y_2) \text { where } x = l_1(a), y = l_2(x_2) \]\[
```
Comment (Keno) [inline exploration/docs/terminology.tex:130]: Ok. The subscript was supposed to indicate indexing, but I agree this is cleaner when compared to the rest of the document.

Comment (Keno) [reply exploration/docs/terminology.tex:130]: `y_2` should be `c` then in this convention

File: exploration/docs/terminology.tex [hunk 3]
```diff
@@ -127,8 +127,8 @@ \section{Optical Constructions}
...
-r_{12}(m_1 \otimes m_2, c) := r_1(m_1, r_2(m_2, c'))
+l_{12}(a) := (m_1 \otimes m_2, y_2) \text { where } (m_1, b) = l_1(a), (m_2, c) = l_2(b) \]\[
```
Comment (mschauer) [inline exploration/docs/terminology.tex]:
```suggestion
l_{12}(a) := (m_1 \otimes m_2, c) \text { where } (m_1, b) = l_1(a), (m_2, c) = l_2(b) \]\[
```

**Review by Keno — APPROVED** [review]

**Pull request merged**